src/shared/components/person/person-listing.tsx

```diff
@@ -11,11 +10,7 @@ import { CakeDay } from "./cake-day";
 
 interface PersonListingProps {
   person: Person;
-  realLink?: boolean;
-  useApubName?: boolean;
-  muted?: boolean;
-  hideAvatar?: boolean;
```

Comment:
These 4 props were only used on the profile page, so they were all rolled into the `profile` prop.